### ISSUE TITLE
Mobs not spawning on main end island 

### ISSUE THREAD

**Issue by youixentoo** [FTBTeam/FTB-Modpack-Issues#6510]
### Modpack

FTB Evolution

### Modpack version

1.6.0

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

Enderman aren't able to naturally spawn on the main end island after killing the dragon and having closed the game. Spawning mobs using mob eggs and /summon also doesn't work. 

### Steps to reproduce

(0. Create world)
1. Kill ender dragon
2. Exit world
3. Restart game
4. Enter world

### Expected behaviour

Mobs should spawn or be able to be spawned in.

### Screenshots

https://youtu.be/EnRIOZM6ZOM

### Additional information

Bug went away after after disabling Savage Ender Dragon.

**Comment by no-name-provided** [comment]
I've also noticed a strange lack of endermen, especially on the central island. Fortunately, they're plentiful in new chunks.

Could be related to https://github.com/someaddons/SavageEnderDragon/issues/58.

**Comment by Gaz492** [comment]
Is this still an issue for you on the latest version of the modpack?

**Comment by youixentoo** [comment]
Yes, so far on every version since 1.6.0 and currently playing on 1.13.0. 

On the main end island:
- mobs will not spawn
- /summon doesn't work
- constructing the wither deletes it

Constructing iron golems and snow golems does work for some reason. Tried summoning an actually additions worm (top of the list of /summon) and it crashed my game (NullPointer).

**Comment by Didas-git** [comment]
I'm having the same issue, i found out that flying extremely high will cause mobs to spawn but they will quickly dispawn right after.

**Comment by unnecessarymb** [comment]
No longer an issue (endermen spawn fine in the latest version)

**Comment by youixentoo** [comment]
We're on version 1.16 now and it still doesn't work for me though

**Comment by unnecessarymb** [comment]
I'll take a look for regressions soon, if I don't find anything your hostile mobcap may just be full. (We could take a look at your save and check.)

**Comment by youixentoo** [comment]
/summon and spawn eggs aren't affected by the mobcap, dreadful dirt also ignores it. Regardless, the mobcap is empty in my testing worlds with like 2 mspt.

**Comment by unnecessarymb** [comment]
Thank you for confirming
